tests/support/util.tcl

```diff
@@ -12,11 +12,9 @@ proc randstring {min max {type binary}} {
         set maxval 52
     }
     while {$len} {
-        set rr [expr {$minval+int(rand()*($maxval-$minval+1))}]
-        if {$type eq {alpha} && $rr eq 92} {
-            set rr 90; # avoid putting '\' char in the string, it can mess up TCL processing
-        }
-        append output [format "%c" $rr]
+        set rr [format "%c" [expr {$minval+int(rand()*($maxval-$minval+1))}]]
+        if {![string is alnum $rr]} {continue}
```

Comment:
First, your change seems to affect all groups (including binary), please fix.. 

Secondly, for alpha, are you sure we want that? 
I'm not sure what did Salvatore choose to add many other (printable) chars, I assume some of them can expose issues, that why I only filtered one char in tat range recently. 
Which char gave you trouble, and why? 
Maybe for your purpose we wanna create a new group, rather than modify the old one?

Reply:
> First, your change seems to affect all groups (including binary), please fix..

Thanks, fixed.
> 
> Secondly, for alpha, are you sure we want that?
> I'm not sure what did Salvatore choose to add many other (printable) chars, I assume some of them can expose issues, that why I only filtered one char in tat range recently.
> Which char gave you trouble, and why?

I wanted to generate client names and not fuck up my `CLIENT LIST` parsing code. I saw there's a function for only alpha(+digits) random str but its implementation didn't make sense.

> Maybe for your purpose we wanna create a new group, rather than modify the old one?

That makes sense, but then I'll modify the old one and create a new group called `printable` which is what the old group should have been called. And fix the code accordingly. Do you want me to do this cleanup?

Reply:
Seems OK, unless there are tons of places using `alpha`, in which case maybe we should find another name rather than rename the current one

Reply:
turns out there are over 20 places...So I'm not sure it's great to do this rename. Don't have a good name for my new one. I'll try `simplealpha`.